program/src/instruction.rs

```diff
@@ -50,6 +50,8 @@ pub enum Instruction {
     /// 3. `[writable]` The aggregator
     /// 4. `[signer]` The aggregator owner
     AddOracle {
+        // Why add an oracle at a specifc index, could instead add at the next empty submission slot.
```

Comment:
I used to by search for pubkey, then I thought that by index might make the program more robust? 
But by search for pubkey maybe more convenience for developer.
Anyway, I'll consider whether to change this.

Reply:
Index would be easier for developers.  If you do decide to keep index the program should explicitly handle cases where the same oracle is added for different indexes

Reply:
Sure.